actions/signed-commits/README.md

```diff
@@ -0,0 +1,252 @@
+# Changesets Release Action
+This action for [Changesets](https://github.com/atlassian/changesets) creates a
+pull request with all of the package versions updated and changelogs updated and
+when there are new changesets on
+[your configured `baseBranch`](https://github.com/changesets/changesets/blob/main/docs/config-file-options.md#basebranch-git-branch-name),
+the PR will be updated. When you're ready, you can merge the pull request and
+you can either publish the packages to npm manually or setup the action to do it
+for you.
+
+## Usage
+
+### Inputs
+
+- publish - The command to use to build and publish packages
+- version - The command to update version, edit CHANGELOG, read and delete
+  changesets. Default to `changeset version` if not provided
+- commit - The commit message to use. Default to `Version Packages`
+- title - The pull request title. Default to `Version Packages`
+- setupGitUser - Sets up the git user for commits as `"github-actions[bot]"`.
+  Default to `true`
+- createGithubReleases - A boolean value to indicate whether to create Github
+  releases after `publish` or not. Default to `true`
+- cwd - Changes node's `process.cwd()` if the project is not located on the
+  root. Default to `process.cwd()`
+
+### Outputs
+
+- published - A boolean value to indicate whether a publishing is happened or
+  not
+- publishedPackages - A JSON array to present the published packages. The format
+  is
+  `[{"name": "@xx/xx", "version": "1.2.0"}, {"name": "@xx/xy", "version": "0.8.9"}]`
+
+### Example workflow:
+
+#### Without Publishing
+
+Create a file at `.github/workflows/release.yml` with the following content.
+
+```yml
+name: Release
+
+on:
+  push:
+    branches:
+      - main
+
+concurrency: ${{ github.workflow }}-${{ github.ref }}
+
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v3
```

Comment:
Ideally all these action references are at least pointing to the latest major version.